compiler/build/scoper.inc.php

```diff
@@ -26,12 +26,6 @@
 	'finders' => [],
 	'files-whitelist' => $stubs,
 	'patchers' => [
-		function (string $filePath, string $prefix, string $content): string {
```

Comment:
this is not needed, git blame does not advised any need/issue in the past with it

if so, please let me know

Reply:
/cc @lookyman Could you remember what was the reason to include this patcher in the original phpstan-compiler? :) Thanks.